FrontEnd/src/components/landing-page/companies/Companies.module.css

```diff
@@ -0,0 +1,47 @@
+.new-companies-main {
+  width: 1512px;
+  height: 1106px;
```

Comment:
It is better not to hardcoding height but to use min or max height

Reply:
Our site isn't responsible for now. And we don't have design to implement styles for responsible components. We are going to work on it after having the design

Reply:
I think that it will work the same way w/o hardcoded height. I know but hardcoded height could introduce unexpected behaviour in future

Reply:
Thank your for feedback. Done